.gimbalrc.yml

```diff
@@ -0,0 +1,135 @@
+    - path: ./build/static/js/tab-weather.*.chunk.js
+      maxSize: 3.5 KB
+    - path: ./build/static/js/tablet.*.chunk.js
+      maxSize: 2 KB
+    - path: ./build/static/js/*.chunk.js
+      maxSize: 50 KB
+    - path: ./build/static/js/runtime*.js
+      maxSize: 3.25 KB
+    - path: ./build/static/media/*.svg
+      maxSize: 1.5 KB
+    - path: ./build/img/App/{nest_manager_analytics,protect_icon,remote_sensor_icon,nst_manager_icon@3x,nst_manager_logo,nest_manager_analytics,info_icon,automation_icon,info_icon}.png
+      maxSize: 275 KB
+    - path: ./build/img/App/{weather_icon,thermostat_icon_2,thermostat_automation_icon,nst_manager_icon@2x}*.png
+      maxSize: 200 KB
+    - path: ./build/img/App/{api_diag_icon,day_calendar_icon,help_ring_icon,leak_icon,login_icon,mode_automation_icon,mode_away_icon,mode_home_icon,nest_manager,open_window}*.png
+      maxSize: 150 KB
+    - path: ./build/img/App/{wiki_icon,vthermostat_icon,update_icon,thermostat_icon,temperature_icon,setpoint_automation_icon,rm_icon,presence_icon,nst_manager_icon,no_icon,nest_dev_away_icon,mode_setpoints_icon,mode_icon,hvac_mode_icon,feedback_icon,fan_control_icon,fan_circulation_icon,diag_icon}*.png
+      maxSize: 100 KB
+    - path: ./build/img/App/help_icon.png
+      maxSize: 350 KB
+    - path: ./build/img/App/*.png
+      maxSize: 50 KB
+    - path: ./build/favicon.ico
+      maxSize: 11 KB
+    - path: ./build/index.html
+      maxSize: 5 KB
+    - path: ./build/
+      maxSize: 35 MB
+
+outputs:
+  cli: false
+  html: artifacts/gimbal.html
+  json: artifacts/gimbal.json
+  markdown: artifacts/gimbal.md
+
+plugins:
+  - plugin: '@modus/gimbal-plugin-axe'
+    thresholds:
+      aria-allowed-role: minor
+      bypass: serious
+      landmark-one-main: moderate
+      meta-viewport: critical
+      page-has-heading-one: moderate
+      region: moderate
+  - plugin: '@modus/gimbal-plugin-source-map-explorer'
+    bundles:
+      - path: '**/main.*.js'
+        thresholds:
+          ../delegated 0 from dll-reference vendor*: 50 B
+          ../external "vendor_*": 75 B
+          ../delegated 65 from dll-reference vendor_*: 50 B
+          ../delegated 1252 from dll-reference vendor_*: 150 B
+          serviceWorker.js: 300 B
+          index.js: 500 B
+          <unmapped>: 150 B
+      - path: '**/*.chunk.js'
+        thresholds:
+          '../**/*.js': 400 KB
+          '**/*.js': 400 KB
+          <unmapped>: 300 B
+      - '!precache-manifest*'
+      - '!service-worker.js'
+      - '!**/runtime*.js'
+  # - '@modus/gimbal-plugin-last-value'
+  # - plugin: '@modus/gimbal-plugin-mysql'
+  #   enabled: ${env:CI, true, false}
+  #   commandPrefix:
+  #     - robodomo-${env:CIRCLE_USERNAME}-${env:CIRCLE_BRANCH}
+  #     - robodomo-${env:CIRCLE_BRANCH}
+  #     - robodomo
+  #   lastValue:
+  #     database: robodomo_gimbal
+  #   host: ${env:GIMBAL_MYSQL_HOST}
+  #   password: ${env:GIMBAL_MYSQL_PASSWORD}
+  #   user: ${env:GIMBAL_MYSQL_USERNAME}
```

Comment:
Commented these out as I don't have access to add env vars. I can provide the creds that I have for the RDS instance the gimbal project itself is using (I have the database already created).